src/main/java/de/uniwue/feature/ProcessStateCollector.java

```diff
@@ -44,6 +44,7 @@ public boolean preprocessingState(String pageId) {
         return true;
     }
 
+
```

Comment:
If possible no unneeded line changes (To prevent more or less unchanged files to appear in the diff)